quickwit-storage/Cargo.toml

```diff
@@ -16,8 +16,10 @@ md5 = '0.7'
 anyhow = '1'
 futures = '0.3'
 bytes = '1'
+serde_json = "1"
 base64 = '0.13'
 tracing = '0.1'
+tantivy = { git= "https://github.com/quickwit-inc/tantivy", rev="c1d1d84"}
```

Comment:
Let's not add tantivy as a deps just for BinarySerializable.

Reply:
tantivy is already a dependency of the whole package though.
What serializer should we use then?

Reply:
It is not a deps of quickwit storage. You only serialize 2 u64?
If so can you just do it directly with rust?